src/python/lib/ensembl/compara/citest/testdb.py

```diff
@@ -24,53 +24,53 @@
 from sqlalchemy import func
 from sqlalchemy.sql.expression import select, text
 
-from ..db import DBConnection
+from ..db import Query, DBConnection
 from ..utils import to_list
 from ._citest import CITestItem
 
 
-class TestDBItem(CITestItem):
+class CITestDBItem(CITestItem):
```

Comment:
Because I forgot to add it. This rename was required because if not, `pytest` by default tries to run every class that starts (or ends) with the word `Test` as a unit test class (and these are not).